KarhooUISDK/Extensions/KarhooSDKExtensions/LocationInfo+Extensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  LocationInfo+Extensions.swift
```

Comment:
Accordin to new guidelines, if extension contains only specific functionality the file name should represent it. In this case it could be something like `LocationInfo+toTripLocationDetails`

Reply:
On the other hand this is extension of Networking SDK type (not iOS SDK) so for now we could agree to keep it this way.

Reply:
It's the extension for the NSDK type. I would keep it generic to avoid renaming it when (if) more extension would be added.